### PR TITLE
Added Swagger Bearer authentication (feature #124)

### LINKED ISSUE #124: add swagger Bearer authentication
Swagger does not support authentication right now.
we need this as some endpoints require logged in users.

This should explain it (not sure):
https://ppolyzos.com/2017/10/30/add-jwt-bearer-authorization-to-swagger-and-asp-net-core/

### PR DESCRIPTION
**Pull request by heijmn**
Updated startup.cs to configure Swagger to use Bearer Authentication

https://github.com/Code2Gether-Discord/JokesOnYou/issues/124

### CODE REVIEW

File: Server/JokesOnYou.Web.Api/Startup.cs [hunk 1]
```diff
@@ -51,6 +52,32 @@ public void ConfigureServices(IServiceCollection services)
             {
                 //appcontext base directory is where the app entry point assembly is (bin folder)
                 config.IncludeXmlComments(Path.Combine(AppContext.BaseDirectory, "JokesOnYou.Web.Api.xml"));
+
+                //add authorization option to Swagger UI
+                config.AddSecurityDefinition("Bearer", new OpenApiSecurityScheme
```
Comment (chrisK00) [inline Server/JokesOnYou.Web.Api/Startup.cs]: Lets move this configuration in to the service extension class with a new method called "ConfigureSwagger" or something like that.

Comment (heijmn) [reply Server/JokesOnYou.Web.Api/Startup.cs]: @chrisK00, moved the logic to ServiceExtensions.cs

**Review by chrisK00 — APPROVED** [review]
Works and looks good. gj

**Review by Niisc — APPROVED** [review]
Noice gj

**Pull request merged**